test/known_issues/test-crypto-authenticated-stream.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable node-core/crypto-check */
```

Comment:
Might not be a bad idea to create a `test/known_issue/.eslintrc.yaml` and disable the test there, but also probably outside the scope of this PR.